database/migrations/2016_02_09_224824_create_competitions_table.php

```diff
@@ -14,9 +14,10 @@ public function up()
     {
         Schema::create('competitions', function (Blueprint $table) {
             $table->increments('id');
-            $table->date('start_date');
-            $table->date('end_date');
+            $table->integer('campaign_id')->unsigned()->index();
             $table->integer('campaign_run_id')->unsigned()->index();
+            $table->date('start_at');
```

Comment:
yeah, I was debating that a bit, cause then in the waiting_room table would it be `signup_start_date` and `signup_end_date`? Also was trying to mimic the naming that the timestamps create... `created_at` & `updated_at`... I can change though if no one likes it :)